src/CommitmentsV2/SFA.DAS.CommitmentsV2.UnitTests/Extensions/DbSetExtensions.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Moq;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+
+namespace SFA.DAS.CommitmentsV2.UnitTests.Extensions
```

Comment:
I think this class may conflict with some of the work on Paul's netcore 3.1 upgrade branch

Reply:
removed store procedures call